api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleServiceTest.java

```diff
@@ -160,13 +165,15 @@ void shouldUpdatePaymentCycleWithEligibilityAndEntitlementDecision() {
         EligibilityAndEntitlementDecision decision = EligibilityAndEntitlementDecision.builder()
                 .dateOfBirthOfChildren(List.of(LocalDate.now().minusYears(1)))
                 .eligibilityStatus(EligibilityStatus.INELIGIBLE)
+                .qualifyingBenefitEligibilityStatus(QualifyingBenefitEligibilityStatus.CONFIRMED)
                 .voucherEntitlement(voucherEntitlement)
                 .build();
 
         paymentCycleService.updatePaymentCycle(paymentCycle, decision);
 
         assertThat(paymentCycle.getId()).isEqualTo(paymentCycle.getId());
         assertThat(paymentCycle.getEligibilityStatus()).isEqualTo(decision.getEligibilityStatus());
+        assertThat(paymentCycle.getQualifyingBenefitEligibilityStatus()).isEqualTo(decision.getQualifyingBenefitEligibilityStatus());
```

Comment:
I know we usually disagree on this type of assertion, but I do think this would be much clearer if it said:

`assertThat(paymentCycle.getQualifyingBenefitEligibilityStatus()).isEqualTo(QualifyingBenefitEligibilityStatus.CONFIRMED);`